src/STCard.ts

```diff
@@ -0,0 +1,52 @@
+import Card from './models/Card/Card';
+import Validation from "./shared/Validation";
+// @ts-ignore
+import template from "./card.html";
+
+class STCard {
+  private _animatedCardTargetContainer: HTMLDivElement;
+  private _card: Card;
+  private _validation: Validation;
+
+  constructor(config: any) {
+    const {animatedCardContainer} = config;
+    this._animatedCardTargetContainer = document.getElementById(animatedCardContainer) as HTMLDivElement;
+    this._animatedCardTargetContainer.innerHTML = template;
+    this._validation = new Validation();
+    Card.ifCardWrapperExist() && (this._card = new Card(config));
```

Comment:
ifCardWrapperExist has been moved to utils hasn't it?

Reply:
yes, actually I deleted most of the implementation stuff and leave only the configuration + core class to start application